### DISCUSSION TITLE
Compiled bindings

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [MahApps/MahApps.Metro#4228] [Ideas]
This would be a great add

[https://github.com/levitali/CompiledBindings](https://github.com/levitali/CompiledBindings)

Could this be integrated into MahApps?

**Comment by punker76** [comment]
Maybe you can ask the maintainer to create a NuGet for this? Then all other can use this too without MahApps.

**Reply by fredericoregateiro** [reply]
You're right, this project is not the best place for this implementation, maybe the ControlzEx.
But i've tested the compiled bindings and so far i don't see any performence improvements, which is strange considering the xamrin.forms improvements
